services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -253,9 +253,10 @@ class ProtectedPathSpec extends HealthCheckSpecification {
     @Unroll
     def "System reroutes #flowDescription flow to more preferable path and ignores protected path when reroute\
  is intentional"() {
-        given: "Two active neighboring switches with two diverse paths at least"
+        // 'and ignores protected path' means that the main path won't changed to protected
```

Comment:
I don't understand the purpose, sorry. If both main and protected paths are changed, then how can you understand whether they swapped or no? Both paths change and you can't understand whether swap happened or not. It doesn't even matter. What if they swapped? I know that this test is old, but still. This should be renamed or removed

Reply:
I believe we need this test
aim of this test is: "verify that system ignores protected path while rerouting and reroutes main/protected paths to more preferable"

Reply:
ok